app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User < ApplicationRecord
   has_secure_password
   has_many :orders
+  has_many :addresses
```

Comment:
We're going to have to start making these destructive if we delete the user by doing a `dependent: :destroy` clause

Reply:
Copy and good catch!